src/lib.rs

```diff
@@ -26,13 +27,20 @@ pub mod usage;
 
 mod response;
 
+static DEEPGRAM_BASE_URL: &str = "https://api.deepgram.com";
+
 /// A client for the Deepgram API.
 ///
 /// Make transcriptions requests using [`Deepgram::transcription`].
 #[derive(Debug, Clone)]
 pub struct Deepgram {
     #[cfg_attr(not(feature = "live"), allow(unused))]
     api_key: String,
```

Comment:
Not the fault of this PR, but the `api_key` is leaked in the `Debug` impl for this struct.

Reply:
Good catch.